dwapi/projects.go

```diff
@@ -51,34 +62,51 @@ func (s *ProjectService) CreateOrReplace(owner, projectid string, body *ProjectC
 	return
 }
 
+// Delete a project and associated data.
 func (s *ProjectService) Delete(owner, projectid string) (response SuccessResponse, err error) {
 	endpoint := fmt.Sprintf("/projects/%s/%s", owner, projectid)
 	headers := s.client.buildHeaders(DELETE, endpoint)
 	err = s.client.request(headers, nil, &response)
 	return
 }
 
+// DownloadFile downloads a file within the project as originally uploaded.
+//
+// Prefer `Query.ExecuteSQL()` or `Query.ExecuteSPARQL()` for retrieving clean and structured data.
 func (s *ProjectService) DownloadFile(owner, projectid, filename string) (response io.Reader, err error) {
 	return s.client.File.Download(owner, projectid, filename)
 }
 
+// DownloadAndSaveFile downloads a file within the project as originally uploaded, and saves the results
+// to a file.
+//
+// Prefer `Query.ExecuteSQL()` or `Query.ExecuteSPARQL()` for retrieving clean and structured data.
 func (s *ProjectService) DownloadAndSaveFile(owner, projectid, filename, path string) (
 	response SuccessResponse, err error) {
 	return s.client.File.DownloadAndSave(owner, projectid, filename, path)
 }
 
+// Download downloads a .zip file containing all files within a project as originally uploaded.
+//
+// Prefer `Query.ExecuteSQL()` or `Query.ExecuteSPARQL()` for retrieving clean and structured data.
 func (s *ProjectService) Download(owner, projectid, filename string) (response io.Reader, err error) {
 	return s.client.File.DownloadDataset(owner, projectid)
 }
 
+// DownloadAndSave downloads a .zip file containing all files within a project as originally
+// uploaded, and saves the results to a file.
+//
+// Prefer `Query.ExecuteSQL()` or `Query.ExecuteSPARQL()` for retrieving clean and structured data.
 func (s *ProjectService) DownloadAndSave(owner, projectid, path string) (response SuccessResponse, err error) {
 	return s.client.File.DownloadAndSaveDataset(owner, projectid, path)
 }
 
+// Liked lists the projects that the currently authenticated user has liked (bookmarked).
```

Comment:
Product team should get this terminology consistent.

Reply:
Bookmarked is really what we're using now but unfortunately, `liked` is still everywhere in the API. I'll keep it as is and I imagine the language will be updated when we move to API version 1.